dataactvalidator/filestreaming/schemaLoader.py

```diff
@@ -86,10 +91,10 @@ def loadCrossRules(filename):
                     targetFileId = validationDb.getFileId(record["target_file"])
                 else:
                     targetFileId = None
-                validationDb.addMultiFieldRule(
+                validationDb.addRule(
                     fileId, record["rule_type"], record["rule_text_one"],
-                    record["rule_text_two"], record["description"],
-                    record["rule_label"], record["rule_timing"], targetFileId)
+                    record["rule_text_two"], record["description"], record["rule_timing"],
+                    record["rule_label"], targetFileId, fileId = fileId)
```

Comment:
Should the crossFile rule load have the same type of error message re: required fields as the regular rules load?